tests/cases/fourslash/aliasMergingWithNamespace.ts

```diff
@@ -0,0 +1,7 @@
+///<reference path="fourslash.ts"/>
+
+////namespace bar { }
+////import bar = bar/**/;
+
+goTo.marker();
+verify.quickInfoExists();
```

Comment:
Yeah I was being lazy. You basically have to run it to find out what the quick info you are expecting is. And the single-test functionality of the fourslash runner is broken :cry: